tasks/discover-granules/schemas/config.json

```diff
@@ -64,6 +69,10 @@
         }
       }
     },
+    "ignoreFilesConfigForDiscovery": {
+      "description": "Overrides this property for all collections.  See same property name at collection level.",
```

Comment:
What do you propose? No description at all? I thought it should at least refer to the collection property where there is a full description, just in case the reader wasn't aware of its existence on the collection level.

Reply:
Sorry, I meant we should put its own description here even if it refers to property for collection.

Reply:
Ah, I see, but I put the reference to the property on the collection not only to indicate that it can be also configured on the collection, but also to avoid duplicating the description here. When we duplicate something, we then run the risk of the duplicates eventually getting out of sync if somebody fails to update all copies in future changes.

Reply:
Okay